Documentation/API.md

```diff
@@ -104,7 +104,7 @@ Creating new operators is also pretty straightforward.
 
 extension NSObject {
 
-    public var rx_deallocated: Observable<Void> {}
+    public var deallocated: Observable<Void> {}
```

Comment:
Not really, `var rx.deallocated` is not valid 😜 
It's more the `extension NSObject` that should be changed.

But anyway, I didn't update documentation yet so this is "normal".